README.rst

```diff
@@ -35,29 +35,26 @@ This product is designed to integrate `redturtle.prenotazioni` package with `des
 Features
 --------
 
-- Can be bullet points
+* Behaviors attached to PrenotazioniFolder c.t.(redturtle.prenotazioni):
```

Comment:
Le varie feature le suddividerei per paragrafi come fatto in altri progetti, così puoi organizzare meglio e scrivere più info

Reply:
@cekk  Fatto